build-system/tasks/visual-diff-wrapper.js

```diff
@@ -0,0 +1,25 @@
+// This file is executed via Puppeteer's page.evaluate on a document to wrap it
+// with an <iframe>, so that we can perform viewport-constrained visual diff
+// tests.
+
+// The texts __WIDTH__ and __HEIGHT__ will be replaced at execution time with
+// the viewport's size from the visual test's config.
+
+const pageContents = document.documentElement.outerHTML;
+
+while (document.head.firstChild) {
+  document.head.removeChild(document.head.firstChild);
+}
+
+while (document.body.firstChild) {
+  document.body.removeChild(document.body.firstChild);
+}
+
+const iframe = document.createElement('iframe');
+iframe.width = __WIDTH__; // eslint-disable-line no-undef
+iframe.height = __HEIGHT__; // eslint-disable-line no-undef
+iframe.srcdoc = pageContents;
+document.body.appendChild(iframe);
+
+document.body.style.margin = '0'; // eslint-disable-line amphtml-internal/no-style-property-setting
+iframe.style.border = '0'; // eslint-disable-line amphtml-internal/no-style-property-setting
```

Comment:
Instead of using `eslint-disable-line`, you could add this file to `.eslintignore`, since I don't think it makes sense to lint it.

In addition, I think there's a check somewhere that requires the AMP copyright at the top of each JS file, that you can skip for this file.

Reply:
Done